src/documentation/runners/jstorm.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: "JStorm Runner"
+permalink: /documentation/runners/jstorm/
+redirect_from: /learn/runners/jstorm/
+---
+# Using the JStorm Runner
+
+The JStorm Runner can be used to execute Beam pipelines using [JStorm](http://jstorm.io/), while providing:
+
+* High throughput and low latency.
+* The at-least-once and exactly-once fault tolerance.
+
+Like a native JStorm topology, user can execute Beam topology on local mode, standalone cluster or jstorm-on-yarn cluster.
+
+The [Beam Capability Matrix]({{ site.baseurl }}/documentation/runners/capability-matrix/) documents the currently supported capabilities of the JStorm Runner.
+
+## JStorm Runner prerequisites and setup
+
+The JStorm runner currently supports JStorm 2.5.0-SNAPSHOT version.
+
+You can add a dependency on the latest version of the JStorm runner by adding to your pom.xml the following:
+```java
+<dependency>
+  <groupId>org.apache.beam</groupId>
+  <artifactId>beam-runners-jstorm</artifactId>
+  <version>{{ site.release_latest }}</version>
+</dependency>
+```
+
+### Deploying JStorm with your application
+
+To run against a Standalone cluster, you can package your program with all beam dependencies into a fat jar, and then submit the topology by following command.
+```
+jstorm jar WordCount.jar org.apache.beam.examples.WordCount --runner=org.apache.beam.runners.jstorm.JStormRunner
+```
+
+For the case that if you don't want to package a fat jar, you can upload the beam dependencies onto all cluster nodes(`$JSTORM_HOME/lib/ext/beam`) first.
```

Comment:
suggest removing "For the case that"  --> "If you don't want to package ..."
capitalize Beam